pages/Useful Utilities/Hyprland-desktop-portal.md

```diff
@@ -82,11 +82,59 @@ See [The Github repo's readme](https://github.com/hyprwm/xdg-desktop-portal-hypr
 {{< hint type=important >}}
 It's recommended to uninstall any other portal implementations to avoid conflicts with the `-hyprland` or `-wlr` ones.
 
-`-kde` and `-gnome` are known to cause issues.
-
-`-kde` is unfortunately a hard dep of `plasma-integration` in Arch Linux, so if using that, you'll need to `pacman -Rnsdd xdg-desktop-portal-kde`.
+`-kde` and `-gnome` portals are known to cause issues.
+
+The `-kde` portal is unfortunately a hard dep of `plasma-integration` in Arch Linux, so to uninstall it,
+you'll need to `pacman -Rnsdd xdg-desktop-portal-kde`, which skips all dependency checks.
```

Comment:
```suggestion
run the command `pacman -Rnsdd xdg-desktop-portal-kde`, which skips all dependency checks.
```
Generally we don't address the readers directly in a wiki, as it's unprofessional. There may be some parts where the text *does* address the readers, but those need cleanup.